mosdns/config-v5.yml

```
@@ -0,0 +1,71 @@
+plugins:
+  - tag: domain_reject
+    type: domain_set
+    args:
+      files:
+        - /etc/mosdns/reject-list.txt
```

let's put it under `/etc/mosdns/rules/`

```suggestion
        - /etc/mosdns/rules/reject-list.txt
```